borg/archive.py

```diff
@@ -650,7 +677,7 @@ def init_chunks(self):
     def report_progress(self, msg, error=False):
         if error:
             self.error_found = True
-        print(msg, file=sys.stderr if error else sys.stdout)
+        logger.log(logging.ERROR if error else logging.WARNING, msg)
```

Comment:
is everything a warning when error is False?
I would have expected rather INFO stuff when error is False.